services/blockstorage/adapter/filesystem/file_system_persistence.go

```diff
@@ -26,14 +26,16 @@ import (
 )
 
 type metrics struct {
-	size *metric.Gauge
+	bytesWritten *metric.Gauge
+	sizeOnDisk   *metric.Gauge
 }
 
 const blocksFilename = "blocks"
 
 func newMetrics(m metric.Factory) *metrics {
 	return &metrics{
-		size: m.NewGauge("BlockStorage.FileSystemSize.Bytes"),
+		bytesWritten: m.NewGauge("BlockStorage.Write.Bytes"),
```

Comment:
why do we need two metrics? what value do we gain from measuring bytes written since startup?

Reply:
perhaps see if there is a spike in storage, maybe a better metric for that would be avg bytes per block, last 1000 blocks